csm/conf/configure.py

```diff
@@ -227,19 +231,19 @@ async def _set_unsupported_feature_info(self):
         """
 
         def get_component_list_from_features_endpoints():
-            Log.info("Get Component List.")
+            Log.info("Config: Getting component list.")
```

Comment:
Comment all the functions and functionalities related to unsupported features

Reply:
@abhijit1patil Is this done?

Reply:
Updated